src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+### 2. Decentralized datastores {#decentralized-datastores}
+
+A blockchain serves as a verifiable data registry; an open, trustless, and decentralized repository of information. The existence of public blockchains, like Ethereum, eliminates the need to store identifiers in centralized registries. 
+
+If anyone needs to confirm the validity of a DID, they can look up the associated public key on the blockchain. This is different from traditional identifiers that can only be authenticated by third parties. 
+
+## How do decentralized identifiers and attestations enable decentralized identity? {#how-decentralized-identifiers-and-attestations-enable-decentralized-identity} 
+
+Decentralized identifiers (DIDs) and attestations are the major building blocks of decentralized identity. Decentralized identity or self-sovereign identity is the idea that identity-related information should be self-controlled, private, and portable. 
+
+In the context of decentralized identity, attestations (also known as [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/)) are tamper-proof, cryptographically verifiable claims made by the issuer. Every attestation or Verifiable Credential an entity (e.g., an organization) issues is associated with their DID. 
+
+Because DIDs are stored on the blockchain, anyone can verify the validity of an attestation by cross-checking the issuer's DID on Ethereum. Essentially, the Ethereum blockchain acts like a global directory that enables the verification of DIDs associated with certain entities.
+
+Decentralized identifiers are the reason attestations are self-controlled and verifiable. Even if the issuer doesn't exist anymore, the holder always has proof of the attestation's provenance and validity. 
+
+DIDs are also crucial to the goal of protect the privacy of personal information through decentralized identity. For instance, if an individual submits proof of an attestation (a driver's license), the verifying party doesn't need to check the validity of information in the proof. The verifier only needs cryptographic guarantees of the attestation's authenticity and the identity of the issuing organization to determine if the proof is valid. 
+
+## Types of attestations in decentralized identity {#types-of-attestations-in-decentralized-identity}
+
+How attestation information is stored and retrieved in an Ethereum-based identity ecosystem is different from traditional identity management. Here is an overview of the various approaches to issuing, storing, and verifying attestations in decentralized identity systems: 
+
+### Off-chain attestations {#off-chain-attestations}
+
+A major concern with storing attestations on-chain is that they contain personally identifiable information (PII) that individuals want to keep private. The transparent nature of the Ethereum blockchain, therefore, makes it unattractive for storing attestations. 
```

Comment:
I wonder if it's worth talking about zk here yet.

Reply:
Yeah, I considered it, but dropped it later because it'd be like forcing readers to take in too much information at a go. I felt that suggesting decentralized identity can prove validity of information without revealing the information was enough (without going down the zk rabbit hole). Methinks if we have a page on ZK proofs, then we can explain how ZK proofs work for identity (and probably link to the decentralized identity page).